ci-operator/jobs/openshift/hypershift/openshift-hypershift-main-presubmits.yaml

```diff
@@ -1,5 +1,73 @@
 presubmits:
   openshift/hypershift:
+  - agent: kubernetes
+    always_run: true
```

Comment:
do we always want to run this test? or do we want to manually invoke it for now?

Reply:
we also likely want
```
optional: true
```

Reply:
Agree, I kept it required since I didn't know if the rehearsal would run it otherwise

Reply:
Fixed